phing/tasks/setup.xml

```diff
@@ -76,6 +76,7 @@
     <chmod mode="0644" failonerror="false" file="${docroot}/sites/default/settings.php"/>
     <echo>Ensuring that blt.settings.php is required by settings.php</echo>
     <exec dir="${docroot}/sites/default" command="grep vendor/acquia/blt/settings/blt.settings.php settings.php || echo 'require DRUPAL_ROOT . &quot;/../vendor/acquia/blt/settings/blt.settings.php&quot;;' >> settings.php" logoutput="true" checkreturn="true" level="${blt.exec_level}"/>
+    <exec dir="${docroot}/sites/default" command="grep trusted_host.settings.php settings.php || echo '// require DRUPAL_ROOT . &quot;/sites/default/settings/trusted_host.settings.php&quot;;' >> settings.php" logoutput="true" checkreturn="true" level="${blt.exec_level}"/>
```

Comment:
I get that, it's not a huge win on its own. Note that the code I added here is commented out, it has no effect on its own.

I just know that we get a lot of questions along the lines of "how do I add custom configuration for x?" The easiest and most intuitive way of demonstrating this is by doing it. Alternatively, we could just add documentation somewhere.